templates/content/field-group-html-element--node--oe-event--group-top.html.twig

```diff
@@ -22,5 +22,6 @@
   </div>
   <div class="ecl-col-12 ecl-col-md-6 ecl-u-mt-l ecl-u-mt-md-none">
     {{ element.extra_field_oe_theme_content_event_details }}
+    {% block oe_event_extra_details %}{% endblock %}
   </div>
 </div>
```

Comment:
We don't prefix block names in templates, as their scope is limited to the template itself.

Let's make this extendible like this: 

```twig
<div id="event-details" class="ecl-row">
  <div class="ecl-col-12 ecl-col-md-6">
    {% block primary_details %}
      {{ element.extra_field_oe_theme_content_event_summary }}
      {{ element.extra_field_oe_theme_content_event_registration_button }}
    {% endblock %}
  </div>
  <div class="ecl-col-12 ecl-col-md-6 ecl-u-mt-l ecl-u-mt-md-none">
    {% block secondary_details %}
      {{ element.extra_field_oe_theme_content_event_details }}
    {% endblock %}
  </div>
</div>
```
Then when extending it we use Twig's [parent function](https://twig.symfony.com/doc/3.x/functions/parent.html):

```twig
{% extends "field-group-html-element--node--oe-event--group-top.html.twig" %}
{% block secondary_details %}
  {{ parent() }}
  ...add here the extra part...
{% endblock %}
```